domain/secretbackend/state/types.go

```diff
@@ -242,3 +244,10 @@ type CloudRow struct {
 	// ModelOwnerUserName holds the name of the user who owns the model.
 	ModelOwnerUserName string `db:"model_owner_user_name"`
 }
+
+// SecretBackendInfo represents a secret backend.
+type SecretBackendInfo struct {
```

Comment:
In ModelSecretBackend struct we use SecretBackendID and ID (for model uuid).
We need to be consistent. I think in that other struct it should be ModelID not just ID